examples/QuickJournalSync/Services/DialogService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using CommunityToolkit.Maui.Alerts;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Views;
+using QuickJournalSync.Views;
+
+namespace QuickJournalSync.Services
+{
+    public static class DialogService
+    {
+        private static Page? MainPage => Application.Current?.MainPage;
```

Comment:
```suggestion
        private static Page MainPage => Application.Current?.MainPage ?? throw new InvalidOperationException("Cannot show dialogs without a Main Page!");
```

And then we don't need the `CheckIfMainPAgeAvailable()` method.